internal/bsr/container.go

```diff
@@ -28,11 +28,39 @@ const (
 	checksumFileName        = "SHA256SUM"
 	sigFileName             = "SHA256SUM.sig"
 	journalFileName         = ".journal"
+	bsrFailedCheckTemplate  = "FAILED: %s"
+)
+
+// Valid BSR checksum status values.
+const (
+	BSRCheckPass BSRChecksumStatus = "OK"
 )
 
 // ContainerType defines the type of container.
 type ContainerType string
 
+// BSRFileName defines the name of a bsr file.
+type BSRFileName string
+
+// BSRChecksumStatus defines the status of the checksum validation
+type BSRChecksumStatus string
+
+// BSRChecksumValidation defines the checksum validation status for a set of files.
+// The key is a file name and the value is the checksum status.
+type BSRChecksumValidation map[BSRFileName]BSRChecksumStatus
```

Comment:
I think we might want the value in this map to be more than just a status. I think there are a few ways this could be done, but in my mind the goal would be to separate the string output from the internal structure. So for example this could become something like:

```go
type FileChecksumValidation struct {
  Filename string
  Passed bool
  Error error
}

type ChecksumValidation map[string]FileChecksumValidation
```

Then the actual output where things like "OK" or "FAILED: <reason>" is displayed could be handled by the cli.